tests/core/repository/mocks.py

```diff
@@ -69,36 +68,20 @@ def _to_entity(self):
 
     @classmethod
     def _from_entity(cls, entity: MockObj):
-        return MockModel(id=entity.id, name=entity.name, version=entity._version)
+        return MockModel(id=entity.id, name=entity.name, version=entity.version)
```

Comment:
```suggestion
        return MockModel(id=entity.id, name=entity.name, version=entity._version)
```
I believe this is a conflict with merged PR https://github.com/Avaiga/taipy-core/pull/663